controllers/machinehealthcheck_controller_test.go

```diff
@@ -496,13 +529,15 @@ var _ = Describe("MachineHealthCheck", func() {
 			}
 
 			// Make sure the status matches.
+			objectKey := util.ObjectKey(mhc)
 			Eventually(func() *clusterv1.MachineHealthCheckStatus {
-				err := testEnv.Get(ctx, util.ObjectKey(mhc), mhc)
+				err := testEnv.Get(ctx, objectKey, mhc)
```

Comment:
I'm not sure this change is strictly necessary?

Reply:
Same